scripts/bench_python_cpp_algos.py

```diff
@@ -292,10 +292,10 @@ def run_all_gap(args):
         },
     ]
 
-    def load_graph(graph_path):
+    def load_graph(graph_path, edge_properties=None, node_properties=None):
         print(f"Running {args.application} on graph: {graph_path}")
         with time_block("read propertyGraph"):
-            graph = PropertyGraph(graph_path)
+            graph = PropertyGraph(graph_path, edge_properties=edge_properties, node_properties=node_properties)
```

Comment:
as I understand the benchmarks, none of them actually use the node properties from the inputs, so we can just set `node_properties=[]` always.

Reply:
If this is true. I am with Donald.

Reply:
Yes you are right